assets/.env

```diff
@@ -14,8 +14,8 @@ POSTGRES_PORT=5432
 DATABASE_URL="postgresql+psycopg2://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}"
 
 ENDPOINT=minio:9000
-MINIO_ACCESS_KEY=minioadmin
```

Comment:
Maybe better to use S3_ACCESS_KEY and S3_SECRET_KEY ?

Reply:
MINIO_ROOT_USER and MINIO_ROOT_PASSWORD are used in .env for docker-compose in order to establish the default user in minio automatically for local usage.
Then, we use those variables to establish connection between our web_app and local minio.
If we work with AWS S3, these particular credentials are not used

Reply:
You may setup `MINIO_ROOT_USER` and `MINIO_ROOT_PASSWORD` for local development using `environments` in `docker-compose.yaml` 

For application make sense to use `S3_ACCESS_KEY` and `S3_SECRET_KEY`

Reply:
I would like to show in code the usage of those variables and discuss it